compat.php

```diff
@@ -46,3 +45,10 @@
 	$gti = new Instant_Articles_Get_The_Image;
 	$gti->init();
 }
+
+// Load support for Playbuzz plugin
+if ( is_plugin_active( 'playbuzz/index.php' ) ) {
```

Comment:
This isn't going to work on VIP (which is probably more our problem, but we should fix it here anyway) but also is vulnerable to changes in the playbuzz plugin itself.

It'd be more reliable to check for the existence of a class or function, if possible.